applications/mne_x_libs/xDisp/helpers/quickcontrolwidget.h

```diff
@@ -129,7 +129,8 @@ class QuickControlWidget : public RoundedEdgesWidget
     *
     * @param [in] qMapChScaling     The pointer to scaling information.
     * @param [in] name              The name to be displayed on the minimize button.
-    * @param [in] slFlags           The flags indicating which tools to display. Scaling is displayed as default. Possible flags are: projections, compensators, view,filter, triggerdetection, modalities, scaling, sphara.
+    * @param [in] pFiffInfo         The fiff info.
+    * @param [in] slFlags           The flags indicating which tools to display. Scaling is displayed as default. Possible flags are: projections, compensators, view, filter, triggerdetection, modalities, scaling, sphara.
     * @param [in] parent            The parent of widget.
     */
     QuickControlWidget(const QMap<qint32, float>& qMapChScaling, const FiffInfo::SPtr pFiffInfo, const QString& name = "", const QStringList& slFlags = QStringList("Scaling"), QWidget *parent = 0);

```

Comment:
Do not use "QWidget *parent = 0", i.e., zero as default pointer, since 0 is also an address; use Q_NULLPTR instead